decoder/vaapidecoder_mpeg2.cpp

```diff
@@ -289,9 +289,6 @@ YamiStatus VaapiDecoderMPEG2::start(VideoConfigBuffer* buffer)
         m_parser->nextStartCode(m_stream.get(), next_code);
         DEBUG("start Next start_code %x", next_code);
         status = processConfigBuffer();
-        if (status == YAMI_DECODE_FORMAT_CHANGE) {
```

Comment:
So afte this patch, the MPEG2 decoder->start() maybe return the DECODE_FORMAT_CHANGE ? But h264 decoder start just get DECODE_FAIL or YAMI_SUCCESS, I think we need to make the start() have a uniform action no matter mpeg2/vc1/vp8/vp9/h264/h265/mjpeg decoder.

@xuguangxin do you have any plan to clear the start() function?